jest/verifyComponent.tsx

```diff
@@ -1,6 +1,6 @@
 /** @jsxRuntime classic */
 /** @jsx jsx */
-import {jsx} from '@emotion/core';
```

Comment:
I believe this may have been working because Emotion 10 still comes in because of Storybook. Tests seem to pass all the same either way, but let's tidy this up.